OCC.Service.Webhost/Repositories/SessionRepository.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using OCC.Data;
+using OCC.Service.Webhost.Services;
+using OCC.Service.Webhost.Tools;
+using Session = OCC.Service.Webhost.Services.Session;
+
+namespace OCC.Service.Webhost.Repositories
+{
+    public class SessionRepository
+    {
+        private readonly OCCDB _dbContext;
+
+        public SessionRepository(OCCDB dbContext)
+        {
+            _dbContext = dbContext;
+        }
+
+        public Session GetSession(int id)
+        {
+            var s = (from x in _dbContext.Sessions.Include("Speaker").Include("Track").Include("Timeslot")
```

Comment:
There seem to be few magic strings we use all over. Let's create a "MagisString" class or something similar where we would define all our magic strings as constants.